uberpoet/moduletree.py

```diff
@@ -78,9 +80,10 @@ def node(f_layer, f_node):
             return ModuleNode('MockLib{}_{}'.format(f_layer, f_node), ModuleNode.LIBRARY)
 
         layers = [[node(l, n) for n in xrange(nodes_per_layer)] for l in xrange(layer_count)]
-        all_nodes = merge_lists(layers)
```

Comment:
this is not needed as its not longer returned below. We create a graph, sort it topologically and return its result instead.